GVFS/GVFS.UnitTests/GVFS.UnitTests.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <Platforms>x64</Platforms>
+    <GenerateProgramFile>false</GenerateProgramFile>

```

Comment:
I will remind myself of the exact issue (and update commit message), but IIRC, this is required because our test project generates an exe - which conflicts with the vstest framework which also automatically injects another static main function. The workaround is to instruct the test framework to not inject another main function.

I think these are relevant links:
Microsoft/vstest/issues/636
Microsoft/vstest/issues/608